### DISCUSSION TITLE
[@types/imapflow] Outdated types

### DISCUSSION THREAD

**Discussion by MainSEL** [DefinitelyTyped/DefinitelyTyped#71437] [Issues with a @types package]
Is it possible to update the types of ImapFlow. I have just noticed that some types no longer match those in the documentation.

For example: ListResponse no longer matches those of @types/imapflow. For example, parent and parentPath are missing

https://imapflow.com/global.html#ListResponse

[a comment by an automated account is not shown]

**Reply by Remscar** [reply]
@MainSEL Please feel free to make a PR with the updated types. I will approve it if you do, as I don't have the time to do this myself.

**Reply by Grayson-code** [reply]
I agree with @Remscar, it would be a good opportunity for you to learn about DT
